spec/features/users_can_edit_a_budget_spec.rb

```diff
@@ -127,21 +130,21 @@
       within(original_row) do
         expect(page).to have_content("£10.00")
         expect(page.all("td", exact_text: "").count).to be 2
-        expect(page).to have_content(I18n.l(Date.today))
+        expect(page).to have_content(I18n.l(Date.yesterday))
```

Comment:
This makes sense to me now, but I don't understand (without putting much thought into it) why it was `Date.today` before, given the time travelling 🤔

Reply:
Before, the time travelling was in a block, so it was returning to the current time outside of the block 🤯